### ISSUE TITLE
Add a LICENSE file

### ISSUE THREAD

**Issue by jhermann** [opensourcedesign/icons#2]
Hi, "… are released under an open source license" means you need to state what that license is. ;)

One of the Creative Commons ones would be the obvious choice.

**Comment by danielhickman** [comment]
I agree, but it should also include the separate ones that are "discovered elsewhere"

**Comment by lewisnyman** [comment]
This will become problematic if each icon discovered elsewhere has different licenses. Some licenses forbid you to re-license them under somewhere else. I think it would be easier just to keep the individual licenses for each group of icons?

**Comment by deadbaed** [comment]
or maybe just license all the icons under the same license, clean and simple

**Comment by jhermann** [comment]
A first step would be to solve the easy case and clearly license "native" contributions, Then you can still think about the other cases.

**Comment by simonv3** [comment]
@bnvk Any preference of license for the things already here?
